pykarma/__init__.py

```diff
@@ -54,6 +54,17 @@ def find(url, fetch_praw=False):
         'kdtoolver': 'b1',
         'q': url
     }
+
+    if subreddit != None:
+        if isinstance(subreddit, (str, unicode)):
```

Comment:
An easier and more legible way to handle this would be -

```python
if isinstance(subreddit, basestring):
    subreddit = [subreddit]
payload['subreddit'] = "|".join(subreddit)
```

Reply:
My bad, 'unicode' was also removed. Making changes.